flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/Table.java

```diff
@@ -161,27 +147,6 @@ TemporalTableFunction createTemporalTableFunction(
      */
     Table as(String field, String... fields);
 
-    /**
-     * Renames the fields of the expression result. Use this to disambiguate fields before joining
-     * to operations.
-     *
```

Comment:
I see a test that uses this function in Scala in OverWindowValidationTest is has .
`Over.partitionBy($"c").orderBy($"rowtime").preceding(2).following($"xx").as($"w"))`